client/.java-version

```diff
@@ -0,0 +1 @@
+22.0.1
```

Comment:
actually, this may cause an issue for you. I built using java 17 eventually

Reply:
What is this one for ? Ohh is this for java version manager ? Btw i am using sdkman and it's on 17.0.7-tem and seems to have built correctly.

Reply:
With 22.0.1-amzn I got

```
@openmsupply-client/android: > Could not resolve all files for configuration ':capacitor-android:androidJdkImage'.
@openmsupply-client/android:    > Failed to transform core-for-system-modules.jar to match attributes {artifactType=_internal_android_jdk_image, org.gradle.libraryelements=jar, org.gradle.usage=java-runtime}.
@openmsupply-client/android:       > Execution failed for JdkImageTransform: /Users/andreievg/Library/Android/sdk/platforms/android-33/core-for-system-modules.jar.
@openmsupply-client/android:          > Error while executing process /Users/andreievg/.sdkman/candidates/java/22.0.1-amzn/bin/jlink with arguments {--module-path /Users/andreievg/.gradle/caches/transforms-3/1988b349790e6cdf8777c1156f6c6868/transformed/output/temp/jmod --add-modules java.base --output /Users/andreievg/.gradle/caches/transforms-3/1988b349790e6cdf8777c1156f6c6868/transformed/output/jdkImage --disable-plugin system-modules}
```

Reply:
yeah, I've removed this file just in case 🙏